lab_1/scripts/build-client.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+echo "Enter source code location"
+read SOURCE_LOCATION
+cd $SOURCE_LOCATION
+npm install
```

Comment:
if we speak about this script usage cases, there is high possibility that it will be used in CI pipeline. So it's better to use "npm ci" instead of "npm install"